README.rst

```diff
@@ -44,7 +44,7 @@ pipeline that makes US energy data easier to access and use programmatically.
 
 Hundreds of gigabytes of valuable data are published by US government agencies, but it's
 often difficult to work with. PUDL takes the original spreadsheets, CSV files, and
-databases and turns them into a unified resource. This allows users to spend more time
+databases and turns them into an unified resource. This allows users to spend more time
```

Comment:
Because `unified` starts with the [`ju` sound](https://en.wiktionary.org/wiki/unified) we actually use `a` to precede it instead of `an`. The more you know!

Reply:
Thanks @jdangerx ! I have fixed the `an unified` issue.